code/clue.js

```diff
@@ -1,59 +1,464 @@
+}
+
+const conservatory = {
+  name: 'Conservatory',
+  image: 'assets/conservatory.png',
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  image: 'assets/kitchen.png',
+}
+
+const study = {
+  name: 'Study',
+  image: 'assets/study.png',
+}
+
+const library = {
+  name: 'Library',
+  image: 'assets/library.png',
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: 'assets/billiard-room.png',
+}
+
+const lounge = {
+  name: 'Lounge',
+  image: 'assets/lounge.png',
+}
+
+const ballroom = {
+  name: 'Ballroom',
+  image: 'assets/ball-room.png',
+}
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.png',
+}
+
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.png',
+}
+
+const livingRoom = {
+  name: 'Living Room',
+  image: 'assets/living-room.png',
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.png',
+}
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.png',
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  image: 'assets/guest-house.png',
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.png',
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// Arrays
 
 const suspects = [
   mrGreen,
+  profPlum,
+  msScarlet,
+  mrsPeacock,
+  colMustard,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
+]
+
+// Variables
 
-const rooms = []
+const killerCard = document.getElementById('killerCard');
+const killerName = document.getElementById('killerName');
+const killerAge = document.getElementById('killerAge');
+const killerImage = document.getElementById('killerImage');
+const killerFavouriteWeapon = document.getElementById('killerFavouriteWeapon');
+const weaponCard = document.getElementById('weaponCard');
+const weaponName = document.getElementById('weaponName');
+const weaponWeight = document.getElementById('weaponWeight');
+const weaponImage = document.getElementById('weaponImage');
+const weaponFavouriteRoom = document.getElementById('weaponFavouriteRoom');
+const roomCard = document.getElementById('roomCard');
+const roomName = document.getElementById('roomName');
+const roomImage = document.getElementById('roomImage');
+const revealedMystery = document.getElementById('mystery');
+const revealButton = document.getElementById('revealButton');
+const resetButton = document.getElementById('resetButton');
+const shuffleButton = document.getElementById('shuffleButton');
+const killerLoader = document.getElementById('killerLoader');
+const weaponLoader = document.getElementById('weaponLoader');
+const roomLoader = document.getElementById('roomLoader');
+
```

Comment:
Nice way of structuring by organising all the variables in one place! Need to try that myself! 😊